vendor/github.com/ethereum/go-ethereum/p2p/discv5/udp.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
-const Version = 55
+const Version = 4
```

Comment:
...this one are important. If other changes improve some service properties and do not break compatibility with the original version, I think they can stay.

Reply:
you are right that other changes don't break compatibility, but i think they are only marginally improving discovery reliability/speed which is not that important for nodes running on PC or VPS.

i will try to get some meaningful metrics from running proxy with and without those changes.

Reply:
Ok, let's do that! These adjustments were only for mobile anyway.